tf_encrypted/keras/layers/normalization.py

```diff
@@ -84,62 +85,83 @@ class BatchNormalization(Layer):
-               axis=3,
-               momentum=0.99,
-               epsilon=1e-3,
-               center=True,
-               scale=True,
-               beta_initializer='zeros',
-               gamma_initializer='ones',
-               moving_mean_initializer='zeros',
-               moving_variance_initializer='ones',
-               beta_regularizer=None,
-               gamma_regularizer=None,
-               beta_constraint=None,
-               gamma_constraint=None,
-               renorm=False,
-               renorm_clipping=None,
-               renorm_momentum=0.99,
-               fused=None, # pylint: disable=unused-argument
-               trainable=False,
-               virtual_batch_size=None,
-               adjustment=None,
-               name=None,
-               **kwargs):
+
+  def __init__(
+      self,
+      axis=3,
+      momentum=0.99,
+      epsilon=1e-3,
+      center=True,
+      scale=True,
+      beta_initializer='zeros',
+      gamma_initializer='ones',
+      moving_mean_initializer='zeros',
+      moving_variance_initializer='ones',
+      beta_regularizer=None,
+      gamma_regularizer=None,
+      beta_constraint=None,
+      gamma_constraint=None,
+      renorm=False,
+      renorm_clipping=None,
+      renorm_momentum=0.99,
+      fused=None,  # pylint: disable=unused-argument
+      trainable=False,
+      virtual_batch_size=None,
+      adjustment=None,
+      name=None,
+      **kwargs
+  ):
     super(BatchNormalization, self).__init__(
-        name=name, trainable=trainable, **kwargs)
+        name=name, trainable=trainable, **kwargs
+    )
 
     self.beta_initializer = initializers.get(beta_initializer)
     self.gamma_initializer = initializers.get(gamma_initializer)
     self.moving_mean_initializer = initializers.get(moving_mean_initializer)
     self.moving_variance_initializer = initializers.get(
-        moving_variance_initializer)
-
-    default_args_check(beta_regularizer,
-                       "beta_regularizer",
-                       "BatchNormalization")
-    default_args_check(gamma_regularizer,
-                       "gamma_regularizer",
-                       "BatchNormalization")
-    default_args_check(beta_constraint,
-                       "beta_constraint",
-                       "BatchNormalization")
-    default_args_check(gamma_constraint,
-                       "gamma_constraint",
-                       "BatchNormalization")
-    default_args_check(renorm,
-                       "renorm",
-                       "BatchNormalization")
-    default_args_check(renorm_clipping,
-                       "renorm_clipping",
-                       "BatchNormalization")
-    default_args_check(virtual_batch_size,
-                       "virtual_batch_size",
-                       "BatchNormalization")
-    default_args_check(adjustment,
-                       "adjustment",
-                       "BatchNormalization")
+        moving_variance_initializer
```

Comment:
another example of a missing comma

Reply:
seems strange that it happens so sporadically

Reply:
with yapf it has to be done by hand